src/Builder/ClassResolver/ClassResolver.php

```diff
@@ -59,7 +60,7 @@ public function resolveClass(string $targetClassName, string $customOrganisation
         }
 
         if (!isset(static::$generatedClassList[$resolvedClassName])) {
-            if (class_exists($resolvedClassName)) {
+            if (ComposerClassLoader::classExist($resolvedClassName)) {
```

Comment:
This class contains major issue. Null will never be returned. Which affects all logic. Is it ok?